src/modules/vendor/components/programs/ProfileContent.vue

```diff
@@ -567,6 +591,21 @@ export default {
     isStepValid (step) {
       return step.areActivitiesValid && !step.activities.some(activity => activity.cards.length === 0);
     },
+    async openSubProgramPublicationModal () {
+      try {
+        const companies = await Companies.list();
+        this.companyOptions = this.formatCompanyOptions(companies);
+        this.subProgramPublicationModal = true;
+      } catch (e) {
+        console.error(e);
```

Comment:
peux etre tu peux réinitialiser `this.companies = []` en cas d'erreur?

Reply:
`this.companyOptions` plutôt ? 😄 

bien vu, done